cwr/schema.go

```diff
@@ -57,21 +57,67 @@ CREATE INDEX registered_work_cwr_id_idx         ON registered_work (cwr_id);
 
 
 CREATE TABLE publisher_control (
-	cwr_id               text NOT NULL,
-	tx_id                text NOT NULL,
-	object_id            text NOT NULL,
-	publisher_sequence_n text NOT NULL,
-	record_type          text NOT NULL
+	cwr_id                            text NOT NULL,
+	tx_id                             text NOT NULL,
+	object_id                         text NOT NULL,
+	publisher_sequence_n              text NOT NULL,
+	transaction_sequence_n            text NOT NULL,
+	record_sequence_n                 text NOT NULL,
+	interested_party_number           text NOT NULL,
+	publisher_name                    text NOT NULL,
+	publisher_unknown_indicator       text NOT NULL,
+	publisher_type                    text NOT NULL,
+	tax_id_number                     text NOT NULL,
+	publisher_ipi_name_number         text NOT NULL,
+	pr_affiliation_society_number     text NOT NULL,
+	submitter_agreement_number        text NOT NULL,
+	pr_ownership_share                text NOT NULL,
+	mr_society                        text NOT NULL,
+	mr_ownership_share                text NOT NULL,
+	sr_society                        text NOT NULL,
+	sr_ownership_share                text NOT NULL,
+	special_agreements_indicator      text NOT NULL,
+	first_recording_refusal_ind       text NOT NULL,
+	publisher_ipi_base_number         text NOT NULL,
+	inter_standard_agreement_code     text NOT NULL,
+	society_assigned_agreement_number text NOT NULL,
+	agreement_type                    text NOT NULL,
+	usa_license_ind                   text NOT NULL,
```

Comment:
Do we really need to index on all of these fields?

Reply:
why not ?

Reply:
The more things we index, the longer it takes and the bigger the resulting index is.

For example, we should only index on `usa_license_ind` if we know we want to query on that field, but I don't see a corresponding query so why index it at all?